tests/search/linguistics_tokens/test.sd

```diff
@@ -0,0 +1,32 @@
+# Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+schema test {
+  document test {
+    field stext type string {
+      indexing: index | summary
+    }
+    field atext type array<string> {
+      indexing: index | summary
+    }
+    field wtext type weightedset<string> {
+      indexing: index | summary
+    }
+  }
+  document-summary tokens {
+    summary stext type string { }
```

Comment:
Summary fields 'stext', 'atext' and 'wtext' do not seem to be used by the test, consider removing.

Reply:
Removed.